gloo/docs/gloo_routing/virtual_services/security/_index.md

```diff
@@ -34,6 +34,85 @@ and providing the following value override:
 | --------------------------------------------------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | global.extensions.extAuth.envoySidecar                    | bool     | Deploy ext-auth as a sidecar to Envoy instances. Communication occurs over Unix Domain Sockets instead of TCP. Default is `false` |
 
-For more details, check out the individual guides for each of the Gloo auth modules:
+#### Configuration overview
+
+{{% notice info %}}
+{{< readfile file="/static/content/extauth_version_info_note" >}}
+{{% /notice %}}
+
+Authentication configuration is defined in [AuthConfig]({{< ref "/api/github.com/solo-io/gloo/projects/gloo/api/v1/enterprise/plugins/extauth/v1/extauth.proto.sk#authconfig" >}}) resources. 
+`AuthConfig`s are top-level resources, which means that if you are running in Kubernetes, they will be stored in a dedicated CRD.
+Here is an example of a simple `AuthConfig` CRD:
+
+
+```yaml
+apiVersion: enterprise.gloo.solo.io/v1
+kind: AuthConfig
+metadata:
+  name: basic-auth
+  namespace: gloo-system
+spec:
+  configs:
+  - basicAuth:
+      apr:
+        users:
+          user:
+            hashedPassword: 14o4fMw/Pm2L34SvyyA2r.
+            salt: 0adzfifo
+      realm: gloo
+```
+
+The resource `spec` consists of an array of `configs` that will be executed in sequence when a request matches the 
+`AuthConfig` (more on how requests are matched to `AuthConfigs` shortly). If any one of these "authentication steps" 
+fails, the request will be denied. In most cases an `AuthConfig` will contain a single `config`.
+
+##### Configuration format
+Once an `AuthConfig` has been created, it can be used to authenticate your `Virtual Services` (or on the lower-level `Proxy` resources). 
+You can define authentication configuration your Virtual Services at three different levels:
+ 
+- on **VirtualHosts**,
+- on **Routes**, and
+- on **WeightedDestinations**.
+
+The configuration format is the same in all three cases and can take one of two forms. It must be specified under the 
+relevant `plugins` attribute (`VirtualHostPlugins`, `RoutePlugins`, or `WeightedDestinationPlugins`) and can take one 
+of two forms. To enable authentication you must reference an existing `AuthConfig`:
+
+```yaml
+(virtualHost|route|weightedDestination)Plugins:
+  extensions:
+    configs:
+      extauth:
+        config_ref:
+          # references the example AuthConfig we defined earlier
+          name: basic-auth
+          namespace: gloo-system
+```
+
+The second form is used to explicitly disable authentication:
+
+```yaml
+(virtualHost|route|weightedDestination)Plugins:
```

Comment:
This is confusing, can you just make it explicit with 3 examples?

Reply:
Done. I did not duplicate the example but instead added a couple of sentences to explain. Let me know if it's ok with you.